src/components/Todo/Todo.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classNames from 'classnames';
+import { typeTodo } from '../../propTypes';
+import './Todo.css';
+
+export class Todo extends React.Component {
+  state = {
+    completed: this.props.completed,
```

Comment:
why do you store it in the state?

Reply:
You can change completed state from the page by toggling the checkbox

Reply:
if anything is changed in the initial array, your state will not be changed (read checklist)